src/Compilers/CSharp/Portable/Lowering/AsyncRewriter/AsyncExceptionHandlerRewriter.cs

```diff
@@ -24,7 +24,7 @@ internal sealed class AsyncExceptionHandlerRewriter : BoundTreeRewriterWithStack
         private readonly SyntheticBoundNodeFactory _F;
         private readonly AwaitInFinallyAnalysis _analysis;
 
-        private AwaitCatchFrame _currentAwaitCatchFrame;
+        private AwaitCatchFrame _parentAwaitCatchFrame, _currentAwaitCatchFrame;
```

Comment:
>private AwaitCatchFrame _parentAwaitCatchFrame, _currentAwaitCatchFrame; [](http://example.com/codeflow?start=8&length=72)

I think it is rather unusual for out code to declare multiple fields or locals in one statement. Makes code less readable, in my opinion.